services/core-web/src/App.tsx

```diff
@@ -12,16 +12,32 @@ export const store = configureStore();
 export type RootState = ReturnType<typeof store.getState>;
 export type AppDispatch = typeof store.dispatch;
 
-const App = () => (
-  <BrowserRouter basename={process.env.BASE_PATH}>
-    <ScrollToTopWrapper>
-      <>
-        <Routes />
-        <ModalWrapper />
-        <DocumentViewer />
-      </>
-    </ScrollToTopWrapper>
-  </BrowserRouter>
-);
+export const useKey = (test: (e: KeyboardEvent) => boolean, ref) => {
```

Comment:
I haven't tested to see if we could use this to add other hotkeys to the page, but I don't see why not!